Makefile

```
@@ -60,7 +60,7 @@ SKIP_RESOURCE_CLEANUP ?= false
 USE_EXISTING_CLUSTER ?= false
 ISOLATED_MODE ?= false
 GINKGO_NOCOLOR ?= false
-GINKGO_LABEL_FILTER ?= short || full
+GINKGO_LABEL_FILTER ?= short || full || local
```

We need to leave this as it was because we don't want the local tests to run.

```suggestion
GINKGO_LABEL_FILTER ?= short || full
```

---

@richardcase perhaps I am not getting it fully, but if it is removed how do we want to pass it to `GINKGO_LABEL_FILTER` when running locally? If you are concerned it would try running local tests in the CI, it should not be the case, because in ach test suite we pass the appropriate filter (local is passed to only vSphere tests as is now)

---

Removed it now